libs/k6/src/common/token.js

```
@@ -0,0 +1,134 @@
+ *    ssn: '12345678901',
+ *    appOwner: '12345678901'
+ * }
+ * @param {number} [iteration=0] - The iteration number for the token generation process.
+ * @param {string} [env='yt01'] - The environment for which the token is being generated.
+ * @returns {Promise} - A promise that resolves to the fetched token.
+ */
+export function getEnterpriseToken(tokenOptions, iteration=0, env='yt01') {  
+    const url = new URL(`https://altinn-testtools-token-generator.azurewebsites.net/api/GetEnterpriseToken`);
+    let extendedOptions = addEnvAndTtlToTokenOptions(tokenOptions, env);
+    for (const key in extendedOptions) {
+        if (extendedOptions.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)